Code/GraphMol/MolStandardize/Normalize.cpp

```diff
@@ -86,22 +98,22 @@ ROMol *Normalizer::normalize(const ROMol &mol) {
   return outmol;
 }
 
-ROMol *Normalizer::normalizeFragment(
+boost::shared_ptr<ROMol> Normalizer::normalizeFragment(
     const ROMol &mol,
     const std::vector<std::shared_ptr<ChemicalReaction>> &transforms) {
-  ROMol *nfrag = new ROMol(mol);
+  boost::shared_ptr<ROMol> nfrag(new ROMol(mol));
```

Comment:
It might be more efficient to use a unique_ptr  here.  I'm trying to get in the habit of doing this over shared_ptrs where appropriate.

Reply:
It would be, but we need the `shared_ptr` because we're going to pass it to `runReactants()` later and creating it as a `shared_ptr` here saves us a molecule copy later.
We need a version of `runReactants` that takes `std::vector<ROMol *>`... that would help. but modifying the reaction API is a different (and bigger) construction site.